cms/static/js/factories/manage_users.js

```diff
@@ -9,7 +9,7 @@ function(_, gettext, ManageUsersAndRoles) {
             var local_messages = _.extend({}, messages);
             local_messages.alreadyMember.title = gettext('Already a course team member');
             local_messages.deleteUser.messageTpl = gettext(
-                'Are you sure you want to delete {email} from the course team for “{container}”?'
+                'Are you sure you want to delete <%= email %> from the course team for “<%= container %>”?'
```

Comment:
@cahrens  edX member
cahrens added a note 5 days ago
Note to reviewers: the users code was early on in my changes. At the time, I desired to get rid of DB's custom interpolation tags. Later on in the review (in LMS code) I gave up on this, so I could change this back to the original if we wish.

Also note that escaping should NOT happen here in the JS. It should always happen "as late as possible", which is in the HTML. Doing it here is likely to cause double-escaping.